src/routes/components/popover/+page.md

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+    import Popover from "$lib/components/Popover.svelte";
+    import {IconMenu, IconAccount, IconExit, IconWallet, IconLightMode, IconDarkMode} from "$lib/icons";
+    let visible = false;
+    let button: HTMLButtonElement | undefined;
+    let visible2 = false;
+    let button2: HTMLButtonElement | undefined;
+    let visible3 = false;
+    let button3: HTMLButtonElement | undefined;
+    let selected1: boolean = true;
+</script>
+
+# Popover
+
+A popover is a floating container that is rendered over the content
+next to an anchor — commonly a button — which initiates its display.
+<br />
+
+# Showcase
```

Comment:
for consistency reason with the other pages, can you move the showcase at the end of the page?

Reply:
@peterpeterparker so the reason I moved the showcase to the top is because if it's at the bottom, the popover will overflow viewport since its placement is always below the anchor. Can we change the popover to account for that, if it's at the bottom? Otherwise the popover will be cut off when you view it.
<img width="393" alt="Screen Shot 2022-10-06 at 11 16 06 AM" src="https://user-images.githubusercontent.com/38238542/194388993-64e52052-b411-4c5a-8450-7a5a37de8e8b.png">

Reply:
Gotcha, you are right, makes sense 👍